config/environments/development.rb

```diff
@@ -75,14 +79,17 @@
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
-  # config.action_view.annotate_rendered_view_with_filenames = true
+  config.action_view.annotate_rendered_view_with_filenames = true
```

Comment:
The Rails default config changed from having this commented to uncommented; we'll accept that change.